libraries/botbuilder-lg/src/evaluator.ts

```diff
@@ -594,6 +597,21 @@ export class Evaluator extends AbstractParseTreeVisitor<any> implements LGTempla
             return standardFunction;
         }
 
+        const pointIndex = name.indexOf('.');
+        if (pointIndex > 0) {
+            const alias = name.substr(0, pointIndex);
+            if (alias in this.templates.namedReferences) {
```

Comment:
Same suggestion regarding `const realTemplate = this.templates.namedReferences[alias]; if (realTemplate) { ... }`